ehrapy/api/encode/__init__.py

```diff
@@ -18,23 +23,23 @@ def encode(ann_data: AnnData, autodetect: bool = False, encodings: Dict[str, Lis
         3. count encoding (https://contrib.scikit-learn.org/category_encoders/count.html)
 
     Args:
-        ann_data: The inital AnnData object
+        data: The initial AnnData or MuData object
```

Comment:
Would be awesome if we could link to the AnnData and MuData class documentation here.

e.g. :class:`~anndata.AnnData